socorro/external/elasticsearch/supersearch.py

```diff
@@ -455,7 +455,7 @@ def update_field(self, **kwargs):
                 index=self.config.elasticsearch_default_index,
                 doc_type='supersearch_fields',
                 id=params['name'],
-            )
+            )['_source']
```

Comment:
Please add a brief comment explaining `[_source]` here (it's easy to miss on a quick visual scan, but it's critical to understand the rest of this operation...).